### DISCUSSION TITLE
Host와 Port의 차이점은 무엇일까요?

### DISCUSSION THREAD

**Discussion by kth990303** [woowacourse-study/2022-http-web-basic-for-all-developer#42] [Q&A]
하나의 IP에서 여러 도메인들을 가질 수 있을 때, host로 도메인을 먼저 조회하고 port번호를 찾는다고 합니다.
그런데 제가 알기론, DNS 네임서버에서 domain 주소와 port가 매핑돼있다고 알고 있습니다.
그렇다면 IP 조회 후에 host를 찾으면 도메인을 알 수 있으므로 port는 필요없는게 되지 않나요?
host를 찾고 port까지 찾아야 하는 이유는 무엇인가요?

**Comment by nbalance97** [comment]
DNS 서버에 도메인 주소와 IP가 매핑되어 있으며, 포트가 매핑되어 있는건 아닌걸로 알고 있습니다😀

https://velog.io/@eunnbi/DNS%EC%99%80-%EC%9E%91%EB%8F%99-%EC%9B%90%EB%A6%AC
